README.md

```diff
@@ -1,25 +1,47 @@
 <!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
 Configures a domain hosted on Route53 to work with AWS Simple Email Service (SES)
 
-Creates the following resources:
+## Prerequisites
+* Ensure [terraform](https://www.terraform.io/intro/getting-started/install.html) is installed
+* Ensure domain is registered in [route53](https://aws.amazon.com/route53/)
+* Ensure an s3 bucket exists and SES has write permissions to it
+* If you have an existing rule set you can skip creating the dependent resource
+* route53 zone id can be obtained by looking up the domain in route53 service
+
+## Getting Started
+1. Import the module called `ses_domain` and update its source property to `github.com/trussworks/terraform-aws-ses-domain` and run `terrafrom init`
```

Comment:
should use the hashicorp module and not github path?!?

Reply:
Yes. Use the registry.

Reply:
> So, I can’t tell if this instruction is correct or not?

It is correct as in the github link will work, I think most folks prefer published packages though. So I ll update this section 👍